libevm/VMFace.h

```diff
@@ -33,6 +33,13 @@ ETH_SIMPLE_EXCEPTION_VM(OutOfGas);
 ETH_SIMPLE_EXCEPTION_VM(OutOfStack);
 ETH_SIMPLE_EXCEPTION_VM(StackUnderflow);
 
+struct VmExecResult
+{
+	VmExecResult(owning_bytes_ref&& _output, bool _revertNeeded) : output(std::move(_output)), revertNeeded(_revertNeeded) {}
+
+	owning_bytes_ref output;
+	bool revertNeeded = false;
```

Comment:
Hm, but then we'll have two ways to signal `VM::exec()` failure - through exceptions and success return flag - which is not good at all.

Are we planning to get rid of throwing exceptions in favor of returned result then?